src/main/java/com/cs56fitnessapp/models/Meal.java

```diff
@@ -1,29 +1,137 @@
 package com.cs56fitnessapp.models;
+import java.util.ArrayList;
+
 
 /**
- * @author Nerissa Hsieh
+ * @author Nerissa Hsieh & Jimwell Castillo
  * Date Created: 10/03/17
  * Date Modified: 10/03/17
  */
 
-public class Meal implements DietFacts {
-    @Override
-    public int getCaloriesIn() {
-        return 0;
-    }
+public class Meal implements DietFacts
+    {
+        @Override
+        public int getCaloriesIn() {
+            return 0;
+        }
 
-    @Override
-    public double getTotalFatIn() {
-        return 0;
-    }
+        @Override
+        public double getTotalFatIn() {
+            return 0;
+        }
+
+        @Override
+        public double getTotalCarbIn() {
+            return 0;
+        }
+
+        @Override
+        public double getProteinIn() {
+            return 0;
+        }
+
+
+        /** Edited Jimwell Castillo
+         * 10.18.17
+         * */
+
+        public class Meals
```

Comment:
As I can see, that was an attempt to represent a collection of meals (not a single Meal) via implementing an array list. Please not that per UML we're implementing a single `Meal` object and what this single meal knows about itself. 
E.g. an instance of a `Meal` class is lunch. Lunch has the information about what food it's composed of. E.g scrambled eggs, coffee, etc. It has the info about its caloric value, how many fat, carb, protein. It also has information about the date and time of that meal. 
This is outlined in UML. 
We're not talking about the collection of meals here. So this array list implementation is incorrect.